packages/office-ui-fabric-react/src/components/Rating/Rating.Props.ts

```diff
@@ -0,0 +1,46 @@
+import * as React from 'react';
+
+/**
+ * Rating component props.
+ */
+export interface IRatingProps extends React.HTMLProps<HTMLElement> {
+  /**
+   * Selected rating
+   */
+  rating?: number;
+
+  /**
+   * Minimum rating, defaults to 1, has to be >= 0
+   */
+  min?: number;
+
+  /**
+   * Maximum rating, defaults to 5, has to be <= min
+   */
+  max?: number;
+
+  /**
+   * Custom icon, defaults to FavoriteStar
+   */
+  icon?: string;
+
+  /**
+   * Size of rating, defaults to small
+   */
+  size?: RatingSize;
+
+  /**
+   * Callback when the rating has changed
+   */
+  onChanged?: (rating: number) => void;
+
+  /**
+   * Optional text for star ratings, will be read by screen readers, defaults to 'Star'.
+   */
+  ratingText?: string;
```

Comment:
In some other components, we've prefixed props like this with 'aria' to indicate that it affects how a component is read and that it has no visible impact. Maybe `ariaIconLabel` if it typically says something like "star"?